spec/03-plugins/38-ai-proxy/02-openai_integration_spec.lua

```diff
@@ -35,26 +35,19 @@ local function wait_for_json_log_entry(FILE_LOG_PATH)
 end
 
 local _EXPECTED_CHAT_STATS = {
-  openai = {
-    instances = {
-      {
-        meta = {
-          plugin_id = '6e7c40f6-ce96-48e4-a366-d109c169e444',
-          provider_name = 'openai',
-          request_model = 'gpt-3.5-turbo',
-          response_model = 'gpt-3.5-turbo-0613',
-        },
-        usage = {
-          completion_token = 12,
-          prompt_token = 25,
-          total_tokens = 37,
-        },
-      },
+  ["ai-proxy"] = {
+    meta = {
+      plugin_id = '6e7c40f6-ce96-48e4-a366-d109c169e444',
+      provider_name = 'openai',
+      request_model = 'gpt-3.5-turbo',
+      response_model = 'gpt-3.5-turbo-0613',
+    },
+    payload = {},
+    usage = {
+      completion_token = 12,
+      prompt_token = 25,
+      total_tokens = 37,
```

Comment:
perhaps update this test to use several plugins? or add a new test for several plugins? e.g. ai-proxy and the transformers plugins

Reply:
I added test for all AI plugins that will send statistic. Thanks